packages/core/src/compatibility/browser.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+const userAgent = navigator.userAgent;
+
+export const Browser = {
```

Comment:
`export const foo = false` is not mutable, but the fields in an `export const` object _are_. I liked how @leebyp had it before.